### PR TITLE
refactor: Update start_new_chain()'s func params

### PR DESCRIPTION
**Pull request by howjmay**

### CODE REVIEW

File: packages/apilib/deploychain.go [hunk 1]
```diff
@@ -47,7 +47,18 @@ func DeployChain(ctx context.Context, par CreateChainParams, stateControllerAddr
 		originatorAddr, stateControllerAddr, par.N, par.T)
 	fmt.Fprint(textout, par.Prefix)
 
-	anchor, err := par.Layer1Client.L2().StartNewChain(ctx, par.OriginatorKeyPair, par.PackageID, nil, suiclient.DefaultGasPrice, suiclient.DefaultGasBudget, par.InitParams.Bytes(), false)
+	anchor, err := par.Layer1Client.L2().StartNewChain(
+		ctx,
+		par.OriginatorKeyPair,
+		par.PackageID,
+		par.InitParams.Bytes(),
+		nil, // FIXME add this field
```
Comment (howjmay) [inline packages/apilib/deploychain.go]: @dessaya do you know what should I use at here

Comment (dessaya) [reply packages/apilib/deploychain.go]: The result of `origin.InitChain()` or so.

If you can't figure it out don't worry, there may still be some missing pieces.

Comment (howjmay) [reply packages/apilib/deploychain.go]: How about my current way adding blockHash and stateRoot in `CreateChainParams`?

Comment (dessaya) [reply packages/apilib/deploychain.go]: It's probably ok, and if not we can always change it later

Comment (howjmay) [reply packages/apilib/deploychain.go]: great

**Review by dessaya — APPROVED** [review]

**Pull request merged**